crates/jet1090/src/main.rs

```diff
@@ -122,6 +127,27 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     }
     options.sources.append(&mut cli_options.sources);
 
+    // example: RUST_LOG=rs1090=DEBUG
+    let env_filter = EnvFilter::from_default_env();
+
+    let subscriber = tracing_subscriber::registry().with(env_filter);
+    match cli_options.log_file.as_deref() {
+        Some("-") if !cli_options.interactive => {
+            // when it's interactive, logs will disrupt the display
+            subscriber.with(fmt::layer().pretty()).init();
+        }
+        Some(log_file) if log_file != "_" => {
```

Comment:
Can you explain the `if log_file != "_"`?
I am sure you don't mean "-" because it's already covered, but then what?

Reply:
(otherwise looks good to me)

Reply:
Sorry about the typo. That should be "-".

A new push should fix it now.